src/main/java/org/folio/dew/batch/authoritycontrol/AuthorityControlToExportFormatMapper.java

```diff
@@ -35,6 +35,10 @@ public AuthorityUpdateHeadingExportFormat convertToExportFormat(AuthorityDataSta
   }
 
   private String convertUserName(Metadata metadata) {
+    if (isBlank(metadata.getStartedByUserFirstName()) && isBlank(metadata.getStartedByUserLastName())) {
```

Comment:
I think we should check only for last name

Reply:
I do not think so because if first name is not blank, then it won't be unknown user. It will be unknown user only when both first and last names are blank/null agree?